res/css/views/context_menus/_RoomGeneralContextMenu.pcss

```diff
@@ -6,6 +6,10 @@
     mask-image: url("$(res)/img/element-icons/roomlist/low-priority.svg");
 }
 
+.mx_RoomGeneralContextMenu_iconMarkAsRead::before {
```

Comment:
This should be done using the SVG icon approach but would warrant too large of a refactor on an untested part of the codebase that I do not want to test right now.